### PR TITLE
Fixed docker volume creation

### PR DESCRIPTION
**Pull request by prosenberg1**
Volumes were being created by using "mkdir" and then mounting the docker volume on those volumes. This has been changed to use docker volume create.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,22 +30,26 @@ build:
 	docker build --target build --network nuodb-net -f dockers/centos/Dockerfile -t nuodb/node-nuodb:$(VERSION)-build .
 
 #:help: test        | Runs the `test` target, building and testing the driver.
+#changed to properly create and mount volumes
 .PHONY: test
 test: build
 	docker volume create cores
-	docker run -it --cap-add=SYS_PTRACE --memory 1g --volume $(CURDIR)/cores:/cores --volume $(CURDIR)/valgrind:/valgrind --name test --rm --network nuodb-net nuodb/node-nuodb:$(VERSION)-build npm run test
+	docker volume create valgrind
```
Comment (adriansuarez) [inline Makefile:37]: Should the `clean` target remove these volumes?

File: build-support/scripts/up [hunk 2]
```diff
@@ -12,15 +12,15 @@ docker run -d --name ad1 --rm \
     -p 48005:48005 \
     -e "NUODB_DOMAIN_ENTRYPOINT=ad1" \
     ${IMG_NAME} nuoadmin
+#changed to properly create and mount volumes
+docker volume create vol1
+
+docker volume create vol2
```
Comment (adriansuarez) [inline build-support/scripts/up:18]: Why don't you call these `backup` and `archive`, since that is what they are used for?

File: Makefile (hunk 1 above)
Comment (prosenberg1) [reply Makefile:37]: It probably should, as we discussed I think the entirety of `make clean` needs a rework so it doesn't completely nuke you docker.

File: build-support/scripts/up (hunk 2 above)
Comment (prosenberg1) [reply build-support/scripts/up:18]: That would be a good change. I stuck with the names that were in here already.

**Pull request closed without merging**